src/Symfony/tests/Integration/OtelSdkBundle/DependencyInjection/config/exporters/config.yaml

```diff
@@ -7,24 +7,17 @@ trace:
   exporters:
     # dsn example
     exporter1: zipkin+http://zipkinhost:1234/path
```

Comment:
DSN-based config has just been deprecated. I'm in favour of removing it from here, unless somebody has a good argument not to...